pkg/storage/pluginstore/store.go

```diff
@@ -11,7 +12,12 @@ import (
 	"go.mongodb.org/mongo-driver/bson"
 )
 
-var _ plugins.StorageProtocol = &Store{}
+type pluginCloser interface {
```

Comment:
Since we aren't using this interface anywhere other than to assert that we are implementing close, how about we remove it and add an extra interface check like so

```go
var _ io.Closer = &Store{}
var _ plugins.StorageProtocol = &Store{}
```

Reply:
Oh, that's a good idea. I changed it to your suggestion